src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -24,14 +25,40 @@ public static class Base64Encoding
         /// </summary>
         /// <param name="value">TBD</param>
         /// <returns>TBD</returns>
-        public static string Base64Encode(this long value) => Base64Encode(value, new StringBuilder()).ToString();
+        public static string Base64Encode(this long value)
+        {
+            return Base64Encode(value, "");
+        }
+
+        public static string Base64Encode(this long value, string prefix)
```

Comment:
After some sleep, I think we may want to mark this version as `internal`, since mis-use of long prefix strings will result in StackOverflowException, and negative `value` inputs will result in an `indexOutOfRange` exception.

I guess the old version has the second hazard, not sure if we should mark this all internal in that case.

Reply:
Yep, you're correct on both points - I'll go fix it.